src/locales/ua.json

```
@@ -0,0 +1,3 @@
+{
+  "Hello": "Hello"
```

Текст в i18n отображает как исходную строку если не находит ключ, то есть если удалить все месседжи и писать просто текст то будет отображаться текст.


Потому давай сделаем все таки ключ.
```suggestion
  {
    main: {
      hello: 'Hello'
    }
  }
```